generate/gen_ext2

```diff
@@ -1,7 +1,6 @@
 #!/bin/sh -ef
 
-SIZE_BLOCKS=$((64 * 1024))
-DAT="$DATA_DIR/ext2.dat"
+cd "$DATA_DIR"
 
-dd if=/dev/zero "of=$DAT" bs=1024 count=$SIZE_BLOCKS
-/sbin/mkfs.ext2 "$DAT"
+dd if=/dev/zero of=ext2.dat bs=1024 count=65536
+mkfs.ext2 "ext2.dat"
```

Comment:
Probably we shouldn't rely on mkfs.ext2 being on the `$PATH`. Actually, it's not there more often that not.

Reply:
Benchmarks are not supposed to be ran on Travis anyway. I have Debian system at my dev box, and here's what I'll get if I'll try to run `mkfs.ext2`:

```
$ mkfs.ext2
zsh: command not found: mkfs.ext2
```

That's because normal user's `$PATH` does not contain `/sbin`. For me, for example, it's just:

    /home/greycat/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games

Reply:
oO, I stand corrected. Will fix it.

Reply:
Fixed.